presto-spi/pom.xml

```diff
@@ -88,7 +88,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <scope>test</scope>
+            <scope>provided</scope>
```

Comment:
We wanna make sure spi doesn't depend on guava. Let's clean up the dependency in the classes that reference guava.

Reply:
Removed now, why do we want this?